### DISCUSSION TITLE
Unable to bring in: spatial-vector-lidar/global/ne_10m_roads/ne_10m_roads.shp

### DISCUSSION THREAD

**Discussion by annieM459** [earthlab-education/Earth-Analytics-2023-01-Intro#507] [Q&A06 Vector Data] [answered]
Hi all! 

I am currently working on the site_map asignment and trying to bring in the spatial-vector-lidar/global/ne_10m_roads/ne_10m_roads.shp . 

I am using this code: 
<img width="741" alt="Screenshot 2023-10-18 at 10 13 58 AM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/0a38ee32-092a-415c-bd75-73633d6dba9a">
Which works for counties, but keep recieving this error for roads...
<img width="1223" alt="Screenshot 2023-10-18 at 10 14 33 AM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/be7ddd4d-d892-4bf8-9974-3b5eefed51a4">
Any ideas?

**Comment by NobodyLivesHere** [comment, marked as answer]
Looks like you've got a redundant /spatial-vector-lidar/ folder in the path.  
Try 
```python
roads = site_map_path / 'global/ne_10m_roads/ne_10m_roads.shp'
```
